frontend/lib/norent/start-account-or-login/ask-phone-number.tsx

```diff
@@ -47,26 +51,34 @@ export const AskPhoneNumber: React.FC<StartAccountOrLoginProps> = (props) => {
           <>
             <PhoneNumberFormField
               {...ctx.fieldPropsFor("phoneNumber")}
-              label="Phone number"
+              label={li18n._(t`Phone number`)}
             />
             <div className="content">
-              <LetterBuilderAccordion question="Why do you need this information?">
-                We’ll use this information to either:
-                <ol className="is-marginless">
-                  <li>Log you into your existing account</li>
-                  <li>Match with a pre-existing account </li>
-                  <li>Sign you up for a new account.</li>
-                </ol>
+              <LetterBuilderAccordion
+                question={li18n._(t`Why do you need this information?`)}
+              >
+                <Trans id="norent.whyIsPhoneNumberNeeded">
```

Comment:
Since this is a long-ish blurb I tried using an `id` here...

Reply:
Nice!